### DISCUSSION TITLE
Cannot enable options for API based filter

### DISCUSSION THREAD

**Discussion by fokcuk** [AdguardTeam/AdGuardHome#5170] [Q&A]
When I try to tick the box for API based filters (adult sites etc) I get an error
Error: control/safebrowsing/enable | empty body with content-type "application/octet-stream" not allowed | 415

**Comment by muchar** [comment]
Same here. Makes Adguard completly unusable.

**Reply by Disqu3-mirOir-qt** [reply]
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options

idk how you talk with api but need set header
